include/libopencmsis/core_cm3.h

```diff
@@ -36,16 +36,35 @@
 
 /* structure as in, for example,
  * DeviceSupport/EnergyMicro/EFM32/efm32tg840f32.h, data from
- * libopencm3/cm3/scb.h. FIXME incomplete. */
+ * libopencm3/cm3/scb.h */
 typedef struct {
-	__IO uint32_t CPUID;
+	__I  uint32_t CPUID;
```

Comment:
This commit needs, as mentioned, much more information abotu what you're doing.  Good to see it in a separate commit, but it's still completely without context.